Sources/Gravatar/Network/Services/ImageDownloadService.swift

```
@@ -41,17 +47,18 @@ public struct ImageDownloadService: ImageDownloader {
         forceRefresh: Bool = false,
         processingMethod: ImageProcessingMethod = .common
     ) async throws -> ImageDownloadResult {
-        if !forceRefresh, let result = cachedImageResult(for: url) {
+        if !forceRefresh, let result = try await cachedImageResult(for: url) {
```

There's no way of properly handling this cache error. It depends on the implementation. Should i go on with the network request or should i let it be thrown? 
This is another reason to keep the cache simple. I don't think we should be throwing from imageCache.getImage(...).